orm/answers.js

```diff
@@ -0,0 +1,186 @@
+const { db } = require('../db');
+const { Answer } = require('../models');
+const UsersORM = require('./users');
+const GamesORM = require('./games');
+const QuestionsORM = require('./questions');
+
+class AnswersORM {
+    constructor() {
+        this.name = 'answers';
+        this.games = 'games';
+        this.users = 'users';
+        this.questions = 'questions';
+        this.msgNoUser = 'This user not exist';
+        this.msgNoCreateGame = 'Dont create game';
+        this.msgNoExistQuestion = 'This question not exist';
+        this.msgNoCreateGame = 'Dont create answer';
+        this.msgNoExistAnsware = 'This answare not exist';
+    }
+
+    async getAllAnswers(gameId, page) {
```

Comment:
Maybe you could standardize the methods by renaming them so that they are the same for every ORM.